.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2.1
+orbs:
+  node: circleci/node@3.0.0
+jobs:
+  deploy_aws:
+    machine:
+      enabled: true
+    steps:
+      - run:
+        name: AWS EC2 deploy
+        command: |
+            # - SSH to the server to deploy
+            # Change to your username
+            EC2_USERNAME=ec2-user
+            # Change to your server's URL or public IP
+            EC2_PUBLIC_DNS=18.221.153.78
+            echo 'using SSH to connect to EC2'
+            ssh -o StrictHostKeyChecking=no $EC2_USERNAME@$EC2_PUBLIC_DNS "cd product-service; git pull; npm i; pm2 restart product-service"
```

Comment:
pm2 won’t work unless you installed it on the server already

Reply:
installed pm2 so merging this one